### ISSUE TITLE
Error building lns-dashboard-plugin

### ISSUE THREAD

**Issue by T00rk** [LiskHQ/lisk-sdk-examples#141]
I followed the example using node `v12.22.9`
```bash
cd lns-dashboard-plugin && npm i && npm run build
```
Output : 
```bash

[...]

> lns-dashboard-plugin@0.1.0 build:node /home/t00rk/Projets/lisk-sdk-examples/tutorials/lisk-name-service/lns-dashboard-plugin
> tsc

node_modules/@liskhq/lisk-framework-http-api-plugin/node_modules/lisk-framework/external_types/pm2-axon-rpc/index.d.ts:6:7 - error TS2300: Duplicate identifier 'ServerExposeCallBack'.

6  type ServerExposeCallBack = (...args: any[]) => void;
        ~~~~~~~~~~~~~~~~~~~~

  node_modules/lisk-framework/external_types/pm2-axon-rpc/index.d.ts:6:7
    6  type ServerExposeCallBack = (...args: any[]) => void;
            ~~~~~~~~~~~~~~~~~~~~
    'ServerExposeCallBack' was also declared here.

node_modules/@liskhq/lisk-framework-http-api-plugin/node_modules/lisk-framework/external_types/pm2-axon-rpc/index.d.ts:8:15 - error TS2300: Duplicate identifier 'Server'.

8  export class Server {
                ~~~~~~

  node_modules/lisk-framework/external_types/pm2-axon-rpc/index.d.ts:8:15
    8  export class Server {
                    ~~~~~~
    'Server' was also declared here.

node_modules/@liskhq/lisk-framework-http-api-plugin/node_modules/lisk-framework/external_types/pm2-axon-rpc/index.d.ts:14:15 - error TS2300: Duplicate identifier 'Client'.

14  export class Client {
                 ~~~~~~

  node_modules/lisk-framework/external_types/pm2-axon-rpc/index.d.ts:14:15
    14  export class Client {
                     ~~~~~~
    'Client' was also declared here.

node_modules/@liskhq/lisk-framework-http-api-plugin/node_modules/lisk-framework/external_types/pm2-axon/index.d.ts:4:1 - error TS6200: Definitions of the following identifiers conflict with those in another file: Socket, SubSocket, SubEmitterSocket, PubSocket, PubEmitterSocket, PushSocket, ReqSocket, RepSocket, PullSocket, ConnectionPort, types

4 declare module 'pm2-axon' {
  ~~~~~~~

  node_modules/lisk-framework/external_types/pm2-axon/index.d.ts:4:1
    4 declare module 'pm2-axon' {
      ~~~~~~~
    Conflicts are in this file.

node_modules/lisk-framework/external_types/pm2-axon-rpc/index.d.ts:6:7 - error TS2300: Duplicate identifier 'ServerExposeCallBack'.

6  type ServerExposeCallBack = (...args: any[]) => void;
        ~~~~~~~~~~~~~~~~~~~~

  node_modules/@liskhq/lisk-framework-http-api-plugin/node_modules/lisk-framework/external_types/pm2-axon-rpc/index.d.ts:6:7
    6  type ServerExposeCallBack = (...args: any[]) => void;
            ~~~~~~~~~~~~~~~~~~~~
    'ServerExposeCallBack' was also declared here.

node_modules/lisk-framework/external_types/pm2-axon-rpc/index.d.ts:8:15 - error TS2300: Duplicate identifier 'Server'.

8  export class Server {
                ~~~~~~

  node_modules/@liskhq/lisk-framework-http-api-plugin/node_modules/lisk-framework/external_types/pm2-axon-rpc/index.d.ts:8:15
    8  export class Server {
                    ~~~~~~
    'Server' was also declared here.

node_modules/lisk-framework/external_types/pm2-axon-rpc/index.d.ts:14:15 - error TS2300: Duplicate identifier 'Client'.

14  export class Client {
                 ~~~~~~

  node_modules/@liskhq/lisk-framework-http-api-plugin/node_modules/lisk-framework/external_types/pm2-axon-rpc/index.d.ts:14:15
    14  export class Client {
                     ~~~~~~
    'Client' was also declared here.

node_modules/lisk-framework/external_types/pm2-axon/index.d.ts:4:1 - error TS6200: Definitions of the following identifiers conflict with those in another file: Socket, SubSocket, SubEmitterSocket, PubSocket, PubEmitterSocket, PushSocket, ReqSocket, RepSocket, PullSocket, ConnectionPort, types

4 declare module 'pm2-axon' {
  ~~~~~~~

  node_modules/@liskhq/lisk-framework-http-api-plugin/node_modules/lisk-framework/external_types/pm2-axon/index.d.ts:4:1
    4 declare module 'pm2-axon' {
      ~~~~~~~
    Conflicts are in this file.


Found 8 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! lns-dashboard-plugin@0.1.0 build:node: `tsc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the lns-dashboard-plugin@0.1.0 build:node script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/t00rk/.npm/_logs/2022-01-29T11_09_33_312Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! lns-dashboard-plugin@0.1.0 build: `npm run build:node && npm run build:web`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the lns-dashboard-plugin@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/t00rk/.npm/_logs/2022-01-29T11_09_33_383Z-debug.log
```

**Comment by Tschakki** [comment]
Hi @T00rk ! Which example were you following exactly? 
Did you try to follow this guide? https://lisk.com/documentation/lisk-sdk/tutorials/lns/2-frontend.html

**Comment by Tschakki** [comment]
Should be fixed by #161
